frontend/src/ui-kit/pages/components/CheckboxPage.tsx

```
@@ -0,0 +1,284 @@
+            <Checkbox
+              label="Indeterminate checkbox"
+              indeterminate={indeterminate}
+              onChange={() => setIndeterminate(!indeterminate)}
+            />
+            <button
+              className="text-sm text-primary-600 hover:underline"
+              onClick={() => {
+                setChecked(!checked);
+                setIndeterminate(false);
+              }}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)